src/stories/OSCALControlProse.stories.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { OSCALReplacedProseWithParameterLabel } from "../components/OSCALControlProse";
+
+export default {
+  title: "Components/Replaced Prose with Parameter Label",
```

Comment:
I know I suggested we should consider separate stories for each component within a file, but I think in this case it's more confusing.

I'd combine into a single `Control Prose` story with no `Default` but:
* Parameters Replaced With Labels
* Parameters Replaced With By-Component Value
* With Parameter Constraints